frontend/src/app/components/open-channel-list/open-channel-list.component.html

```diff
@@ -38,9 +38,9 @@ <h3 matLine>
           matTooltip="Capacity {{
             channel.capacity
               | number: '1.' + info.node.token.decimals + '-' + info.node.token.decimals
-          }} tokens"
+          }} {{ info.node.token.symbol }}"
         >
-          Capacity: {{ channel.capacity | smallNumber }} tokens
+          Capacity: {{ channel.capacity | smallNumber }} {{ info.node.token.symbol }}
```

Comment:
If this should be equal to the above version in the network graph: Is there an opportunity to make this re-usable? E.g. could you add a getter to the `node.token` what-ever-this-is? I have no knowledge about the code-base and if this is possible. But would be cool if.

Reply:
That would be better definitely. But as the above version is attached to the view using d3.js, this would involve some more refactoring that is not worth it here I think.